### DISCUSSION TITLE
PHP SDK non aggiornato per validazione rafforzata?

### DISCUSSION THREAD

**Discussion by dlondero** [fattureincloud/api#232] [Q&A] [answered]
Buongiorno, vi risulta che con le modifiche [2.4.0](https://developers.fattureincloud.it/docs/CHANGELOG#240-2023-01-05) che hanno introdotto validazioni più stringenti per alcune risorse l'SDK PHP non sia più allineato? Nello specifico ho riscontrato che `certified_email` e `vat_number` non possono essere più `null` (permesso dall'sdk e funzionante in precedenza) ma devono essere obbligatoriamente `string`. Confermate?

**Comment by emajo** [comment, marked as answer]
Ciao @dlondero, le nostre SDKs non fanno validazione, tutti i campi sono nullable perché usiamo le stesse classi sia per le richieste che per le risposte, in generale consigliamo di non passare attivamente i campi settati a null, ma, in caso non servissero è meglio ometterli.

Grazie,
Emanuele.
